Automattic/wp-calypso
# Jetpack Pro Dashboard: Add a download button to licenses with downloads

1204137270272763-as-1204696560014071
Requires D112726-code

## Proposed Changes

* Jetpack Pro Dashboard: Add a download button to licenses with downloads (e.g. downloading a third party plugin the license is for).

## Testing Instructions

- Please follow the steps in D112726-code
- Run Jetpack Cloud - http://jetpack.cloud.localhost:3000/partner-portal/licenses
- Make sure you have a license for WooCommerce Bookings
- Expand the license details - you should see a new Download button which when clicked triggers a download for the WooCommerce Bookings plugin.

![Screen Shot 2023-06-21 at 15 59 15](https://github.com/Automattic/wp-calypso/assets/22746396/b615e46b-02ae-4c08-aa6c-57ebd2985cd3)

## Pre-merge Checklist
